client/src/components/Root.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react'
```

Comment:
Please follow following ordering of imports inside your files:

o when importing files, this is the rule you should follow.

the first group of import will be main React packages and third party libs
the second group of import will be actions
the third group of import will be constants, helpers, components that are created inside the app (it is preferred to order this group by the file that import represents)
the fourth group of import will be images
the fifth group of import will be CSS files

Each group will be separated by a new line, so at the end, begging of the file will look like: